packages/react-ui-components/src/components/SchemaForm/Widgets/AdvancedDateSelectDropdownWidget/AdvancedDateSelectDropdownWidget.tsx

```diff
@@ -0,0 +1,51 @@
+import { ICalendarAdvanceSelectedTime, ITimeChange } from '@kelvininc/ui-components';
+import { WidgetProps } from '@rjsf/utils';
+import { get } from 'lodash';
+import React, { useMemo, useState } from 'react';
+import { KvAdvancedDateSelectDropdown, KvFormLabel } from '../../../stencil-generated';
+import { DEFAULT_DROPDOWN_POSITION_OPTIONS } from './config';
+
+const AdvancedDateSelectDropdownWidget = ({ schema, label, id, disabled, required, onChange, uiSchema }: WidgetProps) => {
```

Comment:
Be careful about the props with default values on the `KvAdvancedDateSelectDropdown` component because those values will be lost on re-render. We need to either add those same default values to this component or remove these props from this component in order to not be forwarded.

I noticed this problems with the `size` and `selectedTimezone` props.

Reply:
Yeap that's true. The only thing that I don't like here is that I'm replicating code :confused: